src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -241,6 +241,9 @@ export default {
         }
       } else {
         this.currentWell = { ...this.well(this.position) }
+        this.currentWell.use_adaptive_loading = this.currentWell.loading_target_p1_plus_p2
```

Comment:
You don't need to use a ternary operation here.  You could ensure you've got access to lodash by doing `import _ from 'lodash'` then you can use `_.capitalize(this.currentWell.loading_target_p1_plus_p2 !== '')` which will take the boolean value and change it to a capitalized version of the word.  i.e. True or False.  What you've done is fine though if you prefer it.